<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTrader v0.3</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>493318d6281b31b10ac4c6045f89592c-chart.ico</normaloff>493318d6281b31b10ac4c6045f89592c-chart.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>수동 주문</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>701</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>잔고 및 보유종목현황</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>661</width>
       <height>61</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <row/>
     <column>
      <property name="text">
       <string>예수금(d+2)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총매입</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총평가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총손익</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>총수익률(%)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>추정자산</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>661</width>
       <height>431</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>옵션코드</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매도호가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>매수호가</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>현재 kospi200</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>만기잔여일</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>HV</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>550</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>조회</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>550</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>실시간 조회</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
